src/analysis/inner_product_space/basic.lean

```diff
@@ -222,10 +223,10 @@ lemma inner_neg_right {x y : F} : ⟪x, -y⟫ = -⟪x, y⟫ :=
 by rw [←inner_conj_sym, inner_neg_left]; simp only [ring_hom.map_neg, inner_conj_sym]
 
 lemma inner_sub_left {x y z : F} : ⟪x - y, z⟫ = ⟪x, z⟫ - ⟪y, z⟫ :=
-by { simp [sub_eq_add_neg, inner_add_left, inner_neg_left] }
+by rw [sub_eq_add_neg, inner_add_left, inner_neg_left, tactic.ring.add_neg_eq_sub]
```

Comment:
This isn't intended to be a publically visible lemma. There should be a better name for it somewhere else.

Reply:
yeah, you're right: [sub_eq_add_neg](https://leanprover-community.github.io/mathlib_docs/algebra/group/defs.html#sub_eq_add_neg)